recipes/segemehl/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: segemehl
+  version: 0.2.0
+
+about:
+  home: http://www.bioinf.uni-leipzig.de/Software/segemehl/
+  summary: Short read mapping with gaps
+
+build:
+  number: 0
+
+source:
+  fn: segemehl_0_2_0.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz"
+  sha256: 2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb
+
+requirements:
+  build: 
+    - llvm # [osx]
+    - gcc # [linux]
+    - libgcc # [linux]
+    - zlib
+    - ncurses
+  run: 
+    - libgcc # [linux]
+    - zlib
```

Comment:
One new thing :-)
We are pinning libraries like zlib and ncurses. 
Can you grep in other recipes how this looks like.
- zlib {{CONDA_ZLIB}} or similar.
Sorry in a train right now.

Great work. Now we can put the Galaxy wrapper under testing.